packages/react-components/react-badge/src/components/PresenceBadge/usePresenceBadge.tsx

```diff
@@ -1,38 +1,39 @@
 import * as React from 'react';
 import { slot } from '@fluentui/react-utilities';
 import {
-  presenceAvailableFilled,
-  presenceAvailableRegular,
-  presenceAwayFilled,
-  presenceBlockedRegular,
-  presenceBusyFilled,
-  presenceDndFilled,
-  presenceDndRegular,
-  presenceOfflineRegular,
-  presenceOofRegular,
-  presenceUnknownRegular,
-} from './presenceIcons';
```

Comment:
Now that we're not using the definitions in `presenceIcons.ts`, we should add `@deprecated` comments to them. For example:
```ts
/** @deprecated Use PresenceAwayFilled from @fluentui/react-icons */
export const presenceAwayFilled = ...
```